docs/faq.md

```diff
@@ -0,0 +1,80 @@
+# Frequently seen errors and how to fix them
+
+These are error messages that we see come up often when working with code or
+development workflows that have not been updated to accommodate the
+free-threaded build. We also provide suggested fixes. Please send in pull
+requests to [the repository for this
+document](https://github.com/quansight-labs/free-threaded-compatibility) if you
+run into any confusing free-threading-specific errors that you suspect apply to
+other libraries and aren't covered here.
+
+## Cython compilation errors: `unknown type name '__pyx_vectorcallfunc'`
+
+This happens if you try to build a Cython extension for the free-threaded build
+using the current stable release of Cython (3.0.11 at the time of writing). The
+current stable release of Cython does not support the free-threaded build. You
+must either build Cython from the `master` branch [on
+Github](https://github.com/cython/cython) or use the nightly wheel:
```

Comment:
Maybe it's a good time to add installing 3.1.0b1 from PyPI as a third alternative here?